.travis.yml

```diff
@@ -51,7 +51,7 @@ jobs:
                     | grep -v tests/TestCase.php \
                     && (echo "UNKNOWN FILES DETECTED" && travis_terminate 1) || echo "NO UNKNOWN FILES"
                 - ./check_trailing_spaces.sh || travis_terminate 1
-                - ./dev-tools/tools/phpstan analyse
+                - php -d auto_prepend_file=dev-tools/vendor/autoload.php ./dev-tools/tools/phpstan analyse
```

Comment:
Prepeding the `dev-tools/vendor/autoload.php` is required so that the PHPStan Phar file is able to load `phpstan/phpstan-phpunit` extension but this feels very hacky and I can't say there is no downside. Any better idea?

Reply:
maybe use a dedicated `composer.json` for PHPStan in a nested dir in `dev-tools` ?

Reply:
Perhaps specify using https://github.com/phpstan/phpstan#specify-paths-to-scan?

Reply:
@SpacePossum Do you mean install PHPStan itself using Composer instead of Phive? Otherwise it won't fix the issue I think.

@localheinz I just tried and I doesn't work. As far as I can tell, these options are for parts of the scanned codebase that isn't autoloaded by Composer, not for extensions to PHPStan runtime.

Reply:
indeed, given the issues we have with Phive lately I wouldn't mind switch (back) to composer installs

Reply:
If we switch back to composer installs I'd vote to move all dev dependencies to `dev-tools/composer.json` and merge it to the main `composer.json` from there.